---
layout: post
title:  "Welcome to velocyto!"
date:   2017-10-10 17:10:00
categories: posts
---

# Welcome to the velocyto homepage!

`velocyto` (velox + κύτος, quick cell) is a package for the analysis of expression dynamics in single cell RNA seq data. In particular, it enables estimations of RNA velocities of single cells by distinguishing unspliced and spliced mRNAs in standard single-cell RNA sequencing protocols (see pre-print below for more information).

## Implementations

`veloctyo` currently has two different implementations:

### velocyto.py

The Python implementation includes a [command line tool](http://velocyto.org/velocyto.py/tutorial/index.html#running-the-cli) and an [analysis pipeline](http://velocyto.org/velocyto.py/tutorial/analysis.html#analysis).

See [the detailed documentation](http://velocyto.org/velocyto.py/) for [installation instructions](http://velocyto.org/velocyto.py/install/index.html), [tutorials](http://velocyto.org/velocyto.py/tutorial/index.html) and an overview of [the full API](http://velocyto.org/velocyto.py/fullapi/index.html).

Example Jupyter notebooks are available at the [velocyto-notebooks Github repository](https://github.com/velocyto-team/velocyto-notebooks/tree/master/python).

Report software or documentation issues at the [velocyto.py Github repository](https://github.com/velocyto-team/velocyto.py). If you would like to contribute to development, please contact the authors.

### velocyto.R

#### Installation

The easiest way to install velocyto.R is using `devtools::install_github()` from R:

```R
library(devtools)
install_github("velocyto-team/velocyto.R")
```
You need to have boost (e.g. `sudo apt-get install libboost-dev`) and openmp libraries installed.

#### velocyto.R Tutorials

- [Chromaffin / SMART-seq2](http://pklab.med.harvard.edu/velocyto/notebooks/R/chromaffin.nb.html) - this example shows how to annotate SMART-seq2 reads from bam file and estimate RNA velocity.

- [Dentate Gyrus / loom](http://pklab.med.harvard.edu/velocyto/notebooks/R/DG1.nb.html) - this example shows how to load spliced/unspliced matrices from loom files prepared by [velocyto.py CLI](http://velocyto.org/velocyto.py/tutorial/index.html#running-the-cli), use [pagoda2](https://github.com/hms-dbmi/pagoda2) to cluster/embed cells, and then visualize RNA velocity on that embedding.

- [Mouse BM / dropEst](http://pklab.med.harvard.edu/velocyto/notebooks/R/SCG71.nb.html) - this example shows how to start analysis using dropEst count matrices, which can calculated from inDrop or 10x bam files using [dropEst pipeline](https://github.com/hms-dbmi/dropEst/). It then uses [pagoda2](https://github.com/hms-dbmi/pagoda2) to cluster/embed cells, and then visualize RNA velocity on that embedding.

Report software or documentation issues at the [velocyto.R Github repository](https://github.com/velocyto-team/velocyto.R). If you would like to contribute to development, please contact the authors.

---
## **NOTE**

`velocyto` is currently in alpha release. Some features are still under development. If you find problems with the software or errors in the documentation, report the issue in the appropriate Github repository ([velocyto.py](https://github.com/velocyto-team/velocyto.py) or [velocyto.R](https://github.com/velocyto-team/velocyto.R)). If you would like to contribute please contact the authors.

---

## Pre-print

**RNA velocity in single cells**

Gioele La Manno, Ruslan Soldatov, Hannah Hochgerner, Amit Zeisel, Viktor Petukhov, Maria Kastriti, Peter Lonnerberg, Alessandro Furlan, Jean Fan, Zehua Liu, David van Bruggen, Jimin Guo, Erik Sundstrom, Goncalo Castelo-Branco, Igor Adameyko, Sten Linnarsson, Peter Kharchenko
bioRxiv 206052; doi: [https://doi.org/10.1101/206052](https://doi.org/10.1101/206052)
